### ISSUE TITLE
Adding multiple @IBActions to Card View

### ISSUE THREAD

**Issue by jasan-s** [Yalantis/Koloda#28]
AS of now DidSelectCardAtIndex function fires if anywhere on view is tapped, however I want add couple of buttons on the cardview that if tapped take the data at the current index  and save it to the cloud using a function.  how can I achieve the IBaction objects? 

I essentially want to add two UIimage views that are clickable. I tried to add clear view with UItapgesture in the background view but blocks the swipe action. 

I hope my question made sense.

**Comment by AEugene** [comment]
I think this question is for stackOverflow)
As you would do with tableViewCell. You can make closures as your custom view class properties, you will execute this closures on IBActions.
Then in:
```swift
    func kolodaViewForCardAtIndex(koloda: KolodaView, index: UInt) -> UIView
```
you should set these properties actions.
For example:
```swift
    func kolodaViewForCardAtIndex(koloda: KolodaView, index: UInt) -> UIView {
          var kolodaPhotoView = KolodaPhotoView()
          kolodaPhotoView.rightTapAction = {
              println("Right view was tapped!")
         }
       return kolodaPhotoView
    }
```

**Comment by DanielOrmeno** [comment]
Hello Eugene,
I didn't want to open a new issue for this. I'm currently experiencing the same issue, I've implemented a custom view with a few UIButtons for controlling the behaviour of the card itself. I'm having issues getting the Action outlets of these buttons to fire their respective methods (button click shows in UI as the button changes states but a breakpoint in the action never gets hit). I've tried the following solutions:

Adding delegate gesture recognizers to the draggable views to prevent tap and pan gestures from firing (on touch) when the view is a UIButton. Attaching the debugger I can see the delegate disabling the gesture recognizer for both cases. This doesn't fix my issue, when I click on the button the DidSelectCardAtIndex delegate method **does not** get invoked but a breakpoint in the outlet does not get hit either. 

By inspecting the view layout on xcode, I noticed that the overlay image is always there, I've tried enabling user input through its ImageView but this doesn't fire the outlet action either. 

Any help or pointers on how to implement buttons in the card view would be greatly appreciated! Awesome job btw, loving koloda its a breeze to implement.
